formats.go

```diff
@@ -3,10 +3,10 @@ package audio
 var (
 	// MONO
 
-	// FormatMono22500 is mono 22.5kHz format.
-	FormatMono22500 = &Format{
```

Comment:
This would be a backwards incompatible change. Let's add `22.05khz` as an additional format rather than modifying the existing `FormatMono22500`.

Reply:
Thank you for your review!
Restored 22500 format.